viite-UI/src/view/RoadLayer.js

```diff
@@ -64,21 +64,37 @@
             coordinate = map.getEventCoordinate(event.originalEvent);
-            infoContent.innerHTML = 'Tienumero:&nbsp;' + roadData.roadNumber + '<br>' +
-              'Tieosanumero:&nbsp;' + roadData.roadPartNumber + '<br>' +
-              'Ajorata:&nbsp;' + roadData.trackCode + '<br>' +
-              'AET:&nbsp;' + roadData.startAddressM + '<br>' +
-              'LET:&nbsp;' + roadData.endAddressM + '<br>' +
-              'Hall. luokka:&nbsp;' + displayAdministrativeClass(roadData.administrativeClassId) + '<br>';
+            infoContent.innerHTML =
+                '<div class="popup-line-div">' + '<div>' + 'Tienumero:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.roadNumber + '</div>' + '</div>' +
+                '<div class="popup-line-div">' + '<div>' + 'Tieosanumero:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.roadPartNumber + '</div>' + '</div>' +
+                '<div class="popup-line-div">' + '<div>' + 'Ajorata:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.trackCode + '</div>' + '</div>' +
+                '<div class="popup-line-div">' + '<div>' + 'AET:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.startAddressM + '</div>' + '</div>' +
+                '<div class="popup-line-div">' + '<div>' + 'LET:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.endAddressM + '</div>' + '</div>' +
+                '<div class="popup-line-div">' + '<div>' + 'Hall. luokka:&nbsp;' + '</div>' + '<div class="selectable">' + displayAdministrativeClass(roadData.administrativeClassId) + '</div>' + '</div>';
+            var altShiftPressed = event.originalEvent.shiftKey && event.originalEvent.altKey;
+            if (altShiftPressed) {
+                infoContent.innerHTML += '<hr>';
+                if (!_.isUndefined(roadData.municipalityCode)) {
+                    infoContent.innerHTML += '<div class="popup-line-div">' + '<div>' + 'MunicipalityCode:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.municipalityCode + '</div>' + '</div>';
+                }
+                infoContent.innerHTML +=
+                    '<div class="popup-line-div">' + '<div>' + 'Link&nbsp;id:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.linkId + '</div>' + '</div>' +
+                    '<div class="popup-line-div">' + '<div>' + 'LinearLocation&nbsp;id:&nbsp' + '</div>' + '<div class="selectable">' + roadData.linearLocationId + '</div>' + '</div>' +
+                    '<div class="popup-line-div">' + '<div>' + 'Roadway&nbsp;id:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.roadwayId + '</div>' + '</div>' +
+                    '<div class="popup-line-div">' + '<div>' + 'RoadwayNumber:&nbsp;' + '</div>' + '<div class="selectable">' + roadData.roadwayNumber  + '</div>' + '</div>';
+            }
           }
         }
       }
-      overlay.setPosition(coordinate);
+      if (!(event.originalEvent.altKey))
```

Comment:
Tälle kommentti, lienis joku tyyliin 
// if cannot be selecting text from the hover box (alt down)
?
...vai, miks tää on ny iffin sisällä?